src/_listDefinitions.scss

```diff
@@ -40,6 +40,27 @@ $fd-list-navigation-navigator-bar-right: inset -0.1875rem 0 0 0 var(--sapSelecte
 $fd-list-navigation-navigator-bar-left: inset 0.1875rem 0 0 0 var(--sapSelectedColor);
 $fd-list-navigation-indicator-width: 2.75rem !default;
 
+$fd-list-optional-text-font-family: var(--sapUiFontFamily);
+$fd-list-optional-text-font-size: var(--sapMFontMediumSize);
```

Comment:
`sapMFontMediumSize` is not from the theming repo - https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_fiori_3/css_variables.css

Reply:
updated